lib/rubocop/packs/private.rb

```diff
@@ -146,6 +146,23 @@ def self.validate_failure_mode_strict(package)
 
         errors
       end
+
+      sig { params(args: T.untyped).returns(String) }
+      def self.execute_rubocop(args)
+        with_captured_stdout do
+          RuboCop::CLI.new.run(args)
```

Comment:
I switched from calling out to the system with `bundle exec rubocop` as I measured shelling out took about 5 seconds (loading bundle) but using the CLI took about half a second. This is because we already have the bundle loaded when we call the CLI directly AND we don't have to boot up a new process.